airbyte-integrations/bases/base-java/Dockerfile

```diff
@@ -13,6 +13,9 @@ ENV AIRBYTE_DISCOVER_CMD "/airbyte/javabase.sh --discover"
 ENV AIRBYTE_READ_CMD "/airbyte/javabase.sh --read"
 ENV AIRBYTE_WRITE_CMD "/airbyte/javabase.sh --write"
 
+# Sentry DSN Key
+ENV SENTRY_DSN "https://2a3a1622a2104b8f8d36db4915c956f8@o1114644.ingest.sentry.io/6145965"
```

Comment:
Is it possible to move this DSN to the `.env` file? We usually specify all environment variables there.

Reply:
Done